plugin/evm/atomic/vm/gossiper_atomic_gossiping_test.go

```diff
@@ -20,16 +18,20 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	commonEng "github.com/ava-labs/avalanchego/snow/engine/common"
+
+	"github.com/ava-labs/coreth/plugin/evm/atomic"
+	"github.com/ava-labs/coreth/plugin/evm/vmtest"
 )
 
 // show that a txID discovered from gossip is requested to the same node only if
 // the txID is unknown
 func TestMempoolAtmTxsAppGossipHandling(t *testing.T) {
 	assert := assert.New(t)
 
-	tvm := newVM(t, testVMConfig{})
+	vm := newAtomicTestVM()
+	tvm := vmtest.SetupTestVM(t, vm, vmtest.TestVMConfig{})
 	defer func() {
-		assert.NoError(tvm.vm.Shutdown(context.Background()))
+		assert.NoError(vm.Shutdown(context.Background()))
```

Comment:
Don't we have the convention to use `require` instead of `assert` (especially for error checks)?